packages/protocol-kit/tests/e2e/utils/passkey.ts

```diff
@@ -0,0 +1,47 @@
+import { PasskeyArgType } from '@safe-global/protocol-kit'
+import crypto from 'crypto'
+import { ethers } from 'ethers'
+import { WebAuthnCredentials } from './webauthnShim'
+
+/**
+ * Creates a mock passkey for testing purposes.
+ * @param name User name used for passkey mock
+ * @param webAuthnCredentials WebAuthnCredentials instance used for testing.
+ * @returns Passkey arguments obect with rawId and publicKey.
+ */
+export async function createMockPasskey(
+  name: string,
+  webAuthnCredentials: WebAuthnCredentials
+): Promise<PasskeyArgType> {
+  const passkeyCredential = await webAuthnCredentials.create({
```

Comment:
I think the await is not necessary here

Reply:
Ah yes true! This file is removed, but I [updated the other util file `passkeys.ts` accordingly](https://github.com/safe-global/safe-core-sdk/pull/846/commits/5f5fee6ac7197d0a69c150c56a4c64ad9c07027e).